com.muensterimorbit.game.yaml

```diff
@@ -0,0 +1,17 @@
+id: com.muensterimorbit.game
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: "Muenster im Orbit.x86_64"
+modules:
+  - name: muensterimorbit
+    buildsystem: simple
+    build-commands:
+      - unzip "Muenster-im-Orbit_Linux_v2-0-23.zip" -d extracted
+      - install -Dm755 extracted/Muenster im Orbit.x86_64 /app/bin/Muenster im Orbit.x86_64
```

Comment:
1. this command won't work because the executable has spaces. 
2. what happen with the aarch64 package being build?